src/components/sidebar/MobileTopSidebar.css

```diff
@@ -34,3 +40,11 @@
   font-family: "Pacifico", cursive;
   font-size: 1.8rem;
 }
+
+.mt-sidebar-btm__list {
+  font-size: 1.1rem;
+}
+
+:target {
+  display: grid;
+}
```

Comment:
you can add new line at end of each file for better formatting

Reply:
ok